admin/app/controllers/authenticated/complementary-certifications/complementary-certification/attach-target-profile.js

```diff
@@ -85,7 +85,7 @@ export default class AttachTargetProfileController extends Controller {
         },
       });
 
-      this.router.transitionTo('authenticated.complementary-certifications.list');
+      this.router.transitionTo(`authenticated.complementary-certifications.complementary-certification.details`);
```

Comment:
Pas nécessité des backstick

```suggestion
      this.router.transitionTo('authenticated.complementary-certifications.complementary-certification.details');
```

Reply:
Ah oui bien vu, les restes d'une éxperimentation...